Sources/tuist/tuist.docc/Articles/Tuist/Tuist - Installation.md

```diff
@@ -0,0 +1,47 @@
+# Installation
+
+Learn how to install Tuist in your environment
+
+## Overview
+
+Tuist is a [command-line application](https://en.wikipedia.org/wiki/Command-line_interface) that you need to install in your environment before you can use it. The installation consists of an executable, dynamic frameworks, and a set of resources (e.g., templates). Although you could manually build Tuist from the sources, we recommend using one of the following installation methods to ensure a valid installation.
+
+### Recommended: [rtx](https://github.com/jdx/rtx)
+
+[rtx](https://github.com/jdx/rtx) is an [asdf-compliant](https://asdf-vm.com) alternative to asdf.
+Both tools deterministically manage and activate system dependencies like Tuist.
+If you don't have it installed on your system,
+you can use any of these [installation methods](https://github.com/jdx/rtx#installation).
+Remember to add the suggested line to your shell, which will ensure the right version is activated when you choose a Tuist project directory in your terminal session.
+
+> Note: rtx is recommended over alternatives like [Homebrew](https://brew.sh) because it supports scoping and activating versions to directories, ensuring every environment uses the same version of Tuist deterministically.
+
+Once installed, you can install Tuist through any of the following commands:
+
+
+```bash
+rtx install tuist@3.36.0      # Install a specific version number
+rtx install tuist@3.36.0      # Install a fuzzy version number
```

Comment:
These 2 are the same command 🤔

Reply:
Ouch, good catch. It should be `rtx install tuist@3`